lib/api/core/models/repositories/tokenRepository.js

```diff
@@ -128,15 +128,13 @@ module.exports = function (kuzzle) {
 
   TokenRepository.prototype.verifyToken = function (token) {
     var
-      deferred,
+      deferred = q.defer(),
```

Comment:
This change should be reverted: if `token` is null, then a deferred promise has been created for nothing. So creating _after_ the test is a good thing.